docs/example/README.md

```diff
@@ -106,7 +110,20 @@ In a browser, open up the following urls:
 
 Position the windows so that you can see all four at the same time. The header at the top of the webpage indicates the application and version.
 
-## Deploy SMI Access Control Policies
+## Traffic Policy Modes
+Once the applications are up and running, they can interact with each other using [SMI traffic policies](#smi-traffic-policies) or [permissive mode](#permissive-traffic-policy-mode). In the SMI policy mode, all traffic is denied by default unless explicitly allowed using a combination of SMI access and routing policies. In permissive mode, traffic between application services is automatically configured by `osm-controller`, and SMI policies are not enforced.
```

Comment:
should we ask the user to check the osm-config configmap to see if they are currently using permissive mode?